pigpen-avro/resources/example_data.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
I used this file, along with the schema file, to generate the avro data.

    $ java -jar ~/avro-tools-1.7.6.jar fromjson --codec snappy --schema-file example_schema.avsc example_data.json